samples/web3wallet/src/main/kotlin/com/walletconnect/web3/wallet/ui/common/Namespaces.kt

```diff
@@ -2,12 +2,14 @@ package com.walletconnect.web3.wallet.ui.common
 
 import com.walletconnect.web3.wallet.client.Wallet
 
+//todo change way of fetching chains
 fun getAllMethodsByChainId(namespace: Wallet.Model.Namespace.Proposal, chainId: String): List<String> {
-    return namespace.methods.takeIf { namespace.chains.contains(chainId) } ?: emptyList()
+    return namespace.methods.takeIf { namespace.chains!!.contains(chainId) } ?: emptyList()
```

Comment:
Might be enough to have
```suggestion
    return namespace.methods.takeIf { namespace.chains?.contains(chainId) == true } ?: emptyList()
```

Reply:
Yea pretty much the same